release/pypi/upload_pypi_to_staging.sh

```diff
@@ -25,7 +25,8 @@ pkgs_to_promote=$(\
     curl -fsSL "https://download.pytorch.org/whl/test/${ARCH}/${PACKAGE_NAME}/index.html" \
         | grep "${PACKAGE_NAME}-${PACKAGE_VERSION}${VERSION_SUFFIX}-" \
         | grep "${PLATFORM}" \
-        | cut -d '"' -f2
+        | cut -d '"' -f2 \
+        | cut -d "#" -f1
```

Comment:
This is needed since we attach sha256 sum to the link hence sha256 value needs to be stripped

Reply:
Maybe add this explanation as a comment to the code so that those reading the code can get this additional context?